controllers/awscluster_controller.go

```diff
@@ -80,7 +76,7 @@ func (r *AWSClusterReconciler) Reconcile(req ctrl.Request) (_ ctrl.Result, reter
 		return reconcile.Result{}, nil
 	}
 
-	log = log.WithName(fmt.Sprintf("cluster=%s", cluster.Name))
+	log = log.WithValues("cluster", fmt.Sprintf("%s/%s", cluster.Namespace, cluster.Name))
```

Comment:
We can likely leave off namespace here, since the namespace will be the same as the related `AWSCluster`.